### DISCUSSION TITLE
How to remove the delete method

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#4987] [Help] [answered]
I removed the method in bulkaction but it still shows up. And on the edit page also removed the method with deleting the post, but it is displayed
![3](https://user-images.githubusercontent.com/75489338/202657557-7e36b0f2-3ae9-4244-8673-773b999b222c.JPG)
![4](https://user-images.githubusercontent.com/75489338/202657584-c67ae36b-f01a-48d6-9c6f-45491adb4afa.JPG)
![1](https://user-images.githubusercontent.com/75489338/202657603-6be57923-bcac-4a14-bc7d-e9db77be34da.JPG)
![2](https://user-images.githubusercontent.com/75489338/202657615-518a243e-5460-4982-83a6-6abf6fd8e928.JPG)

**Comment by danharrin** [comment]
`->bulkActions([])`

**Reply by yablochnyi** [reply]
Thank you. How to remove the delete button on the edit page

**Comment by danharrin** [comment, marked as answer]
Return `[]` from `getActions()`
